src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_COUNT = 4;
+    private static final int MAX_RADIUS = 10;
+    private Random rnd = new Random();
+    private final int index = new Random().nextInt(MAX_COUNT);
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        switch (index) {
+            case 0:
+                return new Circle(rnd.nextInt(), colorSupplier.getRandomColor().toString());
```

Comment:
Fixed the way to get the string, but did not fix the return value from the getRandomColor() method of the ColorSupplier class. Make a String return there, then in this class it will not be necessary to call .name() on each color